### ISSUE TITLE
build fails due to lfs quota

### ISSUE THREAD

**Issue by guystreeter** [mozilla-iot/gateway-rpm#5]
```
+ git lfs pull
batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.              
error: failed to fetch some objects from 'https://github.com/mozilla-iot/gateway-rpm.git/info/lfs'

```

**Comment by mrstegeman** [comment]
I’m aware of this and will be fixing it when we build our next release. Thanks for reporting it so I don’t forget!

**Comment by guystreeter** [comment]
Is there an rpm for Fedora 32? That's why I was trying to build it.

**Comment by mrstegeman** [comment]
Fedora 32 was not yet released when we put out 0.12, so there’s no build for it. There will be for our next release. In the meantime, you could either use the Docker image or build and run the gateway manually.

**Comment by mrstegeman** [comment]
Fixed by b04215c
